RaceCars/tests/raceFunctions.test.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  findAverageSpeed,
+  getFasterRacer,
+  findRacersWithEmptyFuel,
+} from "../src/racer-functions";
+import { GasCar } from "../src/gasCar";
+import { SolarCar } from "../src/solarCar";
+
+test("findRacersWithEmptyFuel returns GasCars with empty fuel", () => {
```

Comment:
In this file I would like to see 3 describe blocks, one for each function being tested.